### PR TITLE
Added legacy create/import/publish module tests

### PR DESCRIPTION
**Pull request by Dantemss**
Could potentially run the same tests on more modules or set other things like some metadata.

### CODE REVIEW

File: pages/archive/content.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        return ET.tostring(html, encoding='unicode')
+
+    @property
+    def stable_dict(self):
+        return {**{field: self.dict[field] for field in self._stable_fields},
+                **{'content': self.stable_content}}
+
+    @property
+    def loaded(self):
+        return super().loaded and self.is_element_displayed(*self._json_locator)
```
Comment (m1yag1) [inline pages/archive/content.py]: The page in `pages/archive/base.py` doesn't have a loaded property defined. Do you need to call `super().loaded` on that?

File: pages/archive/content.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        # Remove revision time
+        revised_time = head.find('./{http://www.w3.org/1999/xhtml}meta[@name="revised-time"]')
+        head.remove(revised_time)
+
+        return ET.tostring(html, encoding='unicode')
+
+    @property
+    def stable_dict(self):
+        return {**{field: self.dict[field] for field in self._stable_fields},
```
Comment (m1yag1) [inline pages/archive/content.py]: Nice use of a python 3 only feature.

File: pages/archive/content.py (hunk 1 above)
Comment (Dantemss) [reply pages/archive/content.py]: I could probably skip the call in this case (the default `loaded` method just returns true every time)

File: pages/legacy/module_edit.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    _publish_link_locator = (By.CSS_SELECTOR, 'a[href$="module_publish"]')
+
+    _import_form_locator = (By.CSS_SELECTOR, 'form[action="module_import_form"]')
+    _import_select_locator = (By.CSS_SELECTOR, 'select[name="format"]')
+
+    _content_textarea_locator = (By.ID, 'textarea')
+
+    @property
+    def username(self):
+        return re.search('/Members/([^/]+)/([^/]+)', self.driver.current_url).group(1)
```
Comment (m1yag1) [inline pages/legacy/module_edit.py]: You may want to add this regex as a property and use `re.compile` and then pass it into the `re.search()` function.
```python
_username_regex = re.compile('/Members/([^/]+)/([^/]+)')
@property
    def username(self):
        return re.search(self_username_regex, self.driver.current_url).group(1)
```

File: pages/archive/content.py (hunk 2 above)
Comment (Dantemss) [reply pages/archive/content.py]: Thanks. Took a lot of googling and trying to figure out how to merge dicts properly :)

**Comment by Dantemss** [comment]
@m1yag1 changes have been made

**Pull request merged**